### DISCUSSION TITLE
投个star

### DISCUSSION THREAD

**Discussion by IsBenben** [Meloong-Git/PCL#3858] [讨论]
在B站上看见了PCL有Github，虽然看不懂代码，但是赶紧过来投一个Star（哈哈）。

**Comment by hejiehao** [comment]
正确的 STAR 方式应当是点击下面的按钮
[![](https://img.shields.io/github/stars/Hex-Dragon/PCL2?style=for-the-badge)](https://www.bilibili.com/video/BV1GJ411x7h7)

**Reply by Xiaobei08** [reply]
你小子，~~我被骗了~~
